src/main/java/org/sonarqube/gradle/SonarUtils.java

```diff
@@ -34,6 +34,8 @@
 import javax.annotation.Nullable;
 import org.gradle.api.Project;
 
+import static com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION;
```

Comment:
This is not the right constant. I will fix it when I come to the office.

Reply:
Pushed the right, deprecated constant. Although I honestly think we shouldn't import Android related classes in SonarUtils.